File: documentation/user/en/query/ordering/examples/reference/reference-attribute-natural-multiple-explicit.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					hierarchyWithin(
						"categories",
						attributeEquals("code", "e-readers")
					)
				),
				orderBy(
					referenceProperty(
						"categories",
						pickFirstByEntityProperty(
							attributeNatural("order", DESC)
						),
						attributeNatural("categoryPriority", DESC)
					)
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContentWithAttributes(
							"categories",
							attributeContent("categoryPriority")
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/reference/reference-attribute-natural-multiple-explicit.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("code", "e-readers")
        		)
        	),
        	OrderBy(
        		ReferenceProperty(
        			"categories",
        			PickFirstByEntityProperty(
        				AttributeNatural("order", Desc)
        			),
        			AttributeNatural("categoryPriority", Desc)
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContentWithAttributes(
        				"categories",
        				AttributeContent("categoryPriority")
        			)
        		)
        	)
        )
	)
);